### PR TITLE
[US4]: Wrap images to add setTimeOut and make videos using images

### PR DESCRIPTION
**Pull request by liviaw**
This PR include changes on:
- `` is encapsulated under Image wrapper
- create setTimeOut in each`` element 
- triggers onEnded, which changes files when showing video

### CODE REVIEW

File: src/Components/ErrorToast/ErrorToast.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import "react-toastify/dist/ReactToastify.css";
+
+export const showError: (msg: string) => React.ReactText = (msg: string) =>
+  toast.error(`❌${msg}`, {
+    position: "top-center",
+    autoClose: 5000,
+  });
+
+export const ErrorModal: React.FC = () => (
```
Comment (juliamitchelmore) [inline src/Components/ErrorToast/ErrorToast.tsx]: As PR feedback: this file should have been included in #8 :)

`showError` can get moved to the utils file though, as it's a utility function and not a Component
Also make sure your component name matches the file name - this should be `ErrorToast`

File: src/Components/MediaWrapper/ImageWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+type Props = {
+  file: File;
+  changeImage: () => void;
+  duration: number;
+};
+// future: add media can be optional
+// if no add media, then do ot set timeout
+export const ImageWrapper: React.FC<Props> = ({ file, changeImage, duration }) => {
+  useEffect(() => {
+    const timeout = setTimeout(() => {
```
Comment (juliamitchelmore) [inline src/Components/MediaWrapper/ImageWrapper.tsx:14]: can you use `useTimeout` from 'beautiful-react-hooks' instead?

**Review by juliamitchelmore — APPROVED** [review]
LGTM with minor edits

File: src/Components/MediaWrapper/ImageWrapper.tsx (hunk 2 above)
Comment (liviaw) [reply src/Components/MediaWrapper/ImageWrapper.tsx:14]: I'm not using beautiful-react-hooks anywhere inside the app anymore, would it still be beneficial to use an extra library?

Comment (juliamitchelmore) [reply src/Components/MediaWrapper/ImageWrapper.tsx:14]: ah no, sorry, I thought you still were 😁

**Pull request merged**